src/frontends/pytorch/src/utils.hpp

```diff
@@ -326,6 +326,12 @@ class DummyDecoder : public TorchDecoder {
     virtual std::unordered_map<std::string, ov::Any> get_rt_info() const override {
         FRONT_END_NOT_IMPLEMENTED(get_rt_info);
     }
+    virtual bool has_converter() const override {
+        FRONT_END_NOT_IMPLEMENTED(has_converter);
+    }
+    virtual OutputVector convert(const ov::frontend::NodeContext* context) const override {
```

Comment:
```suggestion
    virtual OutputVector convert(const ov::frontend::NodeContext& context) const override {
```
let us have aligned format with regular conversion rules